src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/FilterVariantTranches.java

```diff
@@ -29,9 +29,13 @@
  * Apply tranche filtering to VCF based on scores from an annotation in the INFO field.  Tranches are specified in
  * percent sensitivity to the variants in the resource files.  For example, if you specify INDEL tranches 98.0 and 99.0
  * using the CNN_2D score the filtered VCF will contain 2 filter tranches for INDELS: CNN_2D_INDEL_Tranche_98.00_99.00
- * and CNN_2D_INDEL_Tranche_99.00_100.00. We expect CNN_2D_INDEL_Tranche_99.00_100.00 to be more sensitive but less precise
- * than CNN_2D_INDEL_Tranche_98.00_99.00, because variants in CNN_2D_INDEL_Tranche_99.00_100.00 have lower scores than
- * variants in the tranche CNN_2D_INDEL_Tranche_98.00_99.00.
+ * and CNN_2D_INDEL_Tranche_99.00_100.00. Variants that scored better than the 98th percentile of variants in the
+ * resources pass through the filter and will have `.` in the filter field. We expect variants in the tranche
```

Comment:
I didn't realize your passing variants are dots.  That's not in accordance with the spec:

"PASS if this position has passed all filters, i.e. a call is made at this position"
" If filters have not been applied, then this field should be set to the missing value."
http://samtools.github.io/hts-specs/VCFv4.2.pdf

Reply:
Marking these as PASS sounds right, as the output currently makes it look like no filters were applied. The simple change would be to call `passFilters()` in an else block in `secondPassApply`. But that implies that all filters pass, which is fine when we're removing old filters, but thats not the default behavior for the tool. So I think the tool either has to always remove old filters, or the (local filter) PASS case needs to preserve the existing failed filters.